docs/sdd.md

```diff
@@ -0,0 +1,72 @@
+# Router Library - Software Design Document (SDD)
+
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+### Metadata
+
+| Item | Description |
+| --- | --- |
+| Maintainer(s) | [GoodluckH](https://github.com/GoodluckH) |
```

Comment:
I think we just want to add the [services team](https://github.com/orgs/Arrow-air/teams/services) as Maintainers for the libraries / services.
@ServiceDog what do you think?

Reply:
[32376a1](https://github.com/Arrow-air/lib-router/pull/10/commits/32376a1818218a0472f03df890e34d76ed744650)